scalableminds/webknossos
# Route to find data

- backend part is finished, new datastore route: `GET http://localhost:9000/data/datasets/Connectomics_Department/ROI2017_wkw/layers/color/findData?token=secretScmBoyToken`

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- click icon in dataset view tab
- if dataset contains data a position with data should be shown

### Issues:
- fixes #3524 

------
